src/components/SatelliteManifestPanel/SatelliteManifestPanel.tsx

```diff
@@ -111,13 +121,13 @@ const SatelliteManifestPanel: FunctionComponent<SatelliteManifestPanelProps> = (
     setDetailsDrawerIsExpanded(false);
   };
 
-  const formatRow = (row: string[], rowIndex: number) => {
+  const formatRow = (row: string[], rowIndex: number, userPermissions: UserPermissions) => {
```

Comment:
Same as above, why'd you add userPermissions as an arg?

Reply:
see above; changed so it's not included.